packages/minio/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE=https://min.io/
+TERMUX_PKG_DESCRIPTION="Multi-Cloud ☁️ Object Storage"
+TERMUX_PKG_LICENSE="AGPL-V3"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2022-12-02T19-19-22Z
+TERMUX_PKG_SRCURL=https://github.com/minio/minio/archive/refs/tags/RELEASE.${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=0cc787305e317951c8c2f6de1ae14938b750817b310b75e2265195ac00244eb1
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_pre_configure() {
+	termux_setup_golang
+
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.46.2
```

Comment:
Please check the SHA256 sum of the downloaded script.

(BTW I can't figure out why this is needed.)

Reply:
> (BTW I can't figure out why this is needed.)

https://github.com/minio/minio/blob/master/Makefile#L22

Reply:
I don't mean that. `golangci/golangci-lint` is just a linter and should not be needed when just compiling.